buildenv/docker/test/Dockerfile

```diff
@@ -65,7 +65,7 @@ RUN apt-get update \
   && rm -rf /var/lib/apt/lists/*
 
 # Install Docker module to run test framework
-RUN echo yes | cpan install JSON Text::CSV XML::Parser
+RUN echo yes | cpan install JSON Text::CSV
```

Comment:
This change seems unrelated: it ought to have been in a separate PR and need not be reverted.

Reply:
In a separate PR I will assume the testing Dockerfile can be deleted altogether (based on the review notes in the original PR review).  

Feel free to change this PR to however you like, it was created automatically through the web-based GIT tooling.